src/Symfony/Bundle/FrameworkBundle/DataCollector/MessengerDataCollector.php

```diff
@@ -63,6 +64,7 @@ public function handle($message, callable $next)
             if (is_object($result)) {
                 $debugRepresentation['result'] = array(
                     'type' => get_class($result),
+                    'object' => $this->cloneVar($result),
                 );
             } else {
```

Comment:
should be cloned as well in case it is an array, as it might contain objects inside the array

Reply:
btw, the template does not seem to use the value in this case when rendering the info, so why storing it ?